src/derivatives.h

```diff
@@ -113,22 +123,23 @@ struct derivatives_nonspatial {
     }
   }
   // Cache and return the first order derivatives using select matrix.
-  matrix<Type> get_sigma_derivative1(std::vector<int> visits) {
+  matrix<Type> get_sigma_derivative1(std::vector<int> visits, matrix<Type> dist) override {
      if (this->sigmad1_cache.count(visits) > 0) {
       return this->sigmad1_cache[visits];
     } else {
       Eigen::SparseMatrix<Type> sel_mat = this->get_sel_mat(visits);
       int n_visists_i = visits.size();
       matrix<Type> ret = matrix<Type>(this->n_theta * n_visists_i, n_visists_i);
       for (int i = 0; i < this->n_theta; i++) {
-        ret.block(i  * n_visists_i, 0, n_visists_i, n_visists_i) = sel_mat * this->sigmad1_cache[this->full_visit].block(i  * this->n_visits, 0, this->n_visits, this->n_visits) * sel_mat.transpose();
+        auto zz = sel_mat * this->sigmad1_cache[this->full_visit].block(i  * this->n_visits, 0, this->n_visits, this->n_visits) * sel_mat.transpose();
```

Comment:
what is the benefit of changing this? just curious

Reply:
oh it is just too wide to be in one line on my screen; actually there are no benefit

Reply:
:-D